scripts/build-plugins/rollup-plugin-build-themes.js

```diff
@@ -68,11 +96,20 @@ function parseBundle(bundle) {
 
 module.exports = function buildThemes(options) {
     let manifest, variants, defaultDark, defaultLight;
+    let isDevelopment = false;
+    const virtualModuleId = '@theme/'
+    const resolvedVirtualModuleId = '\0' + virtualModuleId;
 
     return {
         name: "build-themes",
         enforce: "pre",
 
+        configResolved(config) {
+            if (config.command === "serve") {
+                isDevelopment = true;
+            }
+        },
+
         async buildStart() {
             const { manifestLocations } = options;
             for (const location of manifestLocations) {
```

Comment:
Hmm, perhaps it would be worthwhile to pass the themes as an object mapping the theme name to the location (and the [default theme name](https://gist.github.com/bwindels/16930ddb0bc0f0e14009f8880793f7bf?permalink_comment_id=4128583#gistcomment-4128583)). This is closer to what the config file will look like, and then we can get rid of the lookup functions that load every manifest to find a theme above.